python/graphstorm/dataloading/dataset.py

```diff
@@ -640,8 +642,8 @@ def prepare_data(self, g):
                 if dist_sum(len(test_idx)) > 0:
                     test_idxs[ntype] = test_idx
 
-        print('part {}, train: {}, val: {}, test: {}'.format(get_rank(), num_train,
-                                                             num_val, num_test))
+        logging.debug('part %d, train: %d, val: %d, test: %d',
```

Comment:
I think the size of train/val/test is commonly used by users as it helps them to understand how large the dataset is.
I would suggest we use logging.info()